### PR TITLE
Physics simulation and testing

### PR DESCRIPTION
**Pull request by lysackm**
Physics_simulation.py has a function where you can give the function `get_stats(t)` where t is seconds from the initial state and draw_orbits tests it by drawing the entity positions for several values of t. 

Added gitignore

### CODE REVIEW

File: draw_orbits.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# ax.set_xlim([-258000000000, 258000000000])
+# ax.set_ylim([-258000000000, 258000000000]) 
+
+scatter = ax.scatter([], [])
+
+
+def animate(i):
+    # t is a parameter which varies 
+    # with the frame number 
+    t =  2 * 86400 * i
```
Comment (jjanzenn) [inline draw_orbits.py]: what's the significance of 86400 and 2? Maybe make those constants.

File: draw_orbits.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+
+# non animated daily iteration through drawing
+def daily_points():
+    # get the values of the planets at time t
+
+    t = 0
+    colors = ["r", "b", "g", "c", "m", "y"]
+
+    while t < 31536000:
```
Comment (jjanzenn) [inline draw_orbits.py]: again might be good to make the number a constant

File: draw_orbits.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+        data = sim.get_stats(t)
+        index = 0
+
+        plt.xlim(-258000000000, 258000000000)
+        plt.ylim(-258000000000, 258000000000)
```
Comment (jjanzenn) [inline draw_orbits.py]: same thing here

**Review by jjanzenn — COMMENTED** [review]
looks mostly good. There are some things that might be clearer with constants. Type hinting would be nice for functions, but absolutely not necessary.

File: draw_orbits.py (hunk 1 above)
Comment (lysackm) [reply draw_orbits.py]: 86400 is 1 day in seconds, 2 was because 1 day per second was took to long

**Review by jjanzenn — APPROVED** [review]
lgtm 🚀

**Review by jjanzenn — APPROVED** [review]
:shipit:

**Pull request merged**